CoreConversationView/src/au/gov/asd/tac/constellation/views/conversationview/Conversation.java

```diff
@@ -410,9 +435,9 @@ public void run() {
     private UpdateComponent<GraphReadMethods> senderUpdater = new UpdateComponent<GraphReadMethods>("Senders", LOCK_STAGE) {
         @Override
         protected boolean update(GraphReadMethods graph) {
-            if (graph == null) {
-                return false;
-            }
+//            if (graph == null) {
+//                return false;
+//            }
```

Comment:
Can possibly remove commented out code if it's not needed

Reply:
oh yeah, done